### DISCUSSION TITLE
Change from stale-bot?

### DISCUSSION THREAD

**Discussion by spenserblack** [o2sh/onefetch#662] [Polls]
Looks like stale-bot might not be working anymore (https://github.com/probot/stale/issues/354, https://github.com/probot/stale/issues/349). Also, we haven't gotten any stale updates in a while.

(This doesn't *have* to be a poll, just took this opportunity to try this cool feature :sweat_smile:)

**Comment by spenserblack** [comment]
Hmmm, looks like votes are anonymous, so we won't be able to get a good idea of what *the collaborators and owner* want vs random votes, I guess best to just discuss and ignore the poll :shrug:

**Reply by HallerPatrick** [reply]
I think this depends on the activity of the community. With about 20 issues, this repo is well maintained. I would therefore suggest a switch to `actions/stale` with a low priority or wait until this discussion goes __stale__ 😄

**Reply by o2sh** [reply]
+1 for `actions/stale`

**Comment by spenserblack** [comment]
[GitHub issues now support multiple closed states](https://github.blog/changelog/2022-05-19-the-new-github-issues-may-19th-update/). Would it make sense to close stale issues as "not planned" instead (if possible via the action)?

**Comment by o2sh** [comment]
This would definitely make sense, but I don't think this feature is supported by the [stale](https://github.com/actions/stale) action yet.

**Reply by spenserblack** [reply]
Yeah, just opened https://github.com/actions/stale/issues/744 to ask if there are plans for it.

**Reply by spenserblack** [reply]
Forgot to mention that https://github.com/actions/stale/pull/764 was merged.
